maxwellgithinji/Stack-Overflow-lite
# [Feature #160538517] Mark answer as preffered

**What is this PR supposed to do?**
Implement user's ability to mark answer as preferred

**How can this be manually tested**
* Fire up the API server
* Run the node server
* register and login with your credentials
* post a question
* post an answer to the question'
* navigate to `profile.html` to view the question you have posted
* click on that question, then you see the answers ans mark them as preffered

 **Any background context you want to provide**
The user can only accept answers to his question

**What are the relevant PT stories?**
https://www.pivotaltracker.com/story/show/160538517

**Screenshots**
![bandicam 2018-09-20 04-15-52-709](https://user-images.githubusercontent.com/11288990/45790320-e9472f80-bc8c-11e8-8cf1-d14892819a0b.jpg)
![bandicam 2018-09-20 09-56-41-612](https://user-images.githubusercontent.com/11288990/45801115-90db5680-bcbb-11e8-9db4-d4a42430bb76.jpg)



**Questions:**
